docs/user/openstack/known-issues.md

```diff
@@ -14,6 +14,13 @@ In order to validate your OpenStack infrastructure prior to installation or upgr
 
 Since the installer requires the *Name* of your external network and Red Hat Core OS image, if you have other networks or images with the same name, it will choose one randomly from the set. This is not a reliable way to run the installer. We highly recommend that you resolve this with your cluster administrator by creating unique names for your resources in openstack.
 
+## Soft-anti-affinity
+
+A long-standing OpenStack Compute issue prevents the **soft anti-affinity**
+policy to be correctly applied when instances are created in parallel. When
+machines must land in distinct Compute hosts, explicitly set
+`serverGroupPolicy` to `anti-affinity` for the machine pool.
```

Comment:
Perhaps add a comment that the IPI installer works around this problem by creating instances sequentially.

Reply:
Creating instances sequentially didn't prove to be effective in 100% of the cases, so I'd prefer not giving that sort of guarantees

Reply:
OK, just out of curiosity, do you have specific example in mind where creating the instances sequentially caused them to end up on the same hypervisor?

Reply:
Just realized, what you're saying here contradicts the first sentence.

Reply:
I have rephrased. Can you please check that it makes more sense now?

Reply:
Yes, thanks a lot :+1: